src/Infrastructure/Common/ContainerBuilder.php

```diff
@@ -108,6 +113,26 @@ private static function loadTranslatorConfiguration(SymfonyContainerBuilder $con
         $containerBuilder->setDefinition('translator', $translatorDefinition);
     }
 
+    private static function loadLoggerConfiguration(SymfonyContainerBuilder $container)
+    {
+        $fileSystemHelper = new FileSystemHelper();
+
+        $logsDir = __DIR__.DIRECTORY_SEPARATOR.'..'.DIRECTORY_SEPARATOR.'..'.DIRECTORY_SEPARATOR.'..'.DIRECTORY_SEPARATOR.'var'.DIRECTORY_SEPARATOR.'logs';
+
+        $logsFile = $logsDir.DIRECTORY_SEPARATOR.'migration_log.txt';
+
+        if (!$fileSystemHelper->fileExists($logsFile)) {
+            mkdir($logsDir);
+            file_put_contents($logsFile, '');
+        }
+
+        $loggerDefinition = new Definition(Logger::class, ['app']);
+        $loggerDefinition->addMethodCall('pushHandler', [new StreamHandler($logsFile)]);
```

Comment:
Adding a second StreamHandler with warning as level would be great. Having warnings and errors in a specific file (error.log) allow to find them easily.